tidy_data/tidy_data.md

```diff
@@ -0,0 +1,737 @@
+
+<h4>Song Data</h4>
+
+| id | artist | track | time |
+| -- | ----- | ----- | ----- |
+| 1 | 2 Pac   | Baby Don't Cry | 4:22 |  
+| 2 | 2Ge+her | The Hardest Part of ... | 3:15 |
+| 3 | 3 Doors Down | Kryptonite | 3:53 |
+
+
+<h4>Ranking Data</h4>
+
+| id | date | week | rank |
+| ----  | ---  | ---  | ---- |
+| 1 | 2000-02-26 | 1 | 87 |
+| 1 | 2000-03-04 | 2 | 82 |
+| 1 | 2000-03-11 | 3 | 72 |
+| 2 | 2000-09-02 | 1 | 91 |
+| 2 | 2000-09-09 | 2 | 87 |
+| 2 | 2000-09-16 | 3 | 92 |
+| 3 | 2000-04-08 | 1 | 81 |
+| 3 | 2000-04-15 | 2 | 70 |
+| 3 | 2000-04-22 | 3 | 68 |
+
+This means that we only have to correct a mispelled artist name or incorrect track duration in one place, rather than multiple locations.  This is especially important in SQL databases.  Still, this is more an issue of data storage and ease of updating, rather than cleaning up data for computation.  Consider this advice completely optional!
+
+### One type in multiple tables
+
+Occasionally we also have data that treat the same topic but are stored in separate tables.  For example, if you're participating in a multi-site study, you may have data that is structured identically but arrive in your inbox in separate files. Logically, you could imagine "stacking" this data vertically, and perhaps adding a new column that gives the data collection site:
+
+<h4>Clinic A</h4>
+
+| subject_id | dob | abc_score | xyz_score |
+| ---------- | --- | --------- | --------- |
+| 4123691    | 2009-08-12 | 87 | 4 |
+| 4296120    | 2005-10-04 | 82 | 5 |
+| 2934218    | 2008-03-28 | 70 | 4 |
+| 2098821    | 2008-12-10 | 88 | 3 |
+
+<h4>Clinic B</h4>
+
+| subject_id | dob | abc_score | xyz_score |
+| ---------- | --- | --------- | --------- |
+| 3396811    | 2010-12-29 | 78 | 4 |
+| 0129681    | 2010-01-02 | 80 | 3 |
+| 9766728    | 2007-07-10 | 77 | 3 |
+| 6382922    | 2008-03-20 | 90 | 5 |
+
+<h4>Combined Data</h4>
+
+| subject_id | dob | abc_score | xyz_score | collection_site |
+| ---------- | --- | --------- | --------- | -- |
+| 4123691    | 2009-08-12 | 87 | 4 | A |
+| 4296120    | 2005-10-04 | 82 | 5 | A |
+| 2934218    | 2008-03-28 | 70 | 4 | A |
+| 2098821    | 2008-12-10 | 88 | 3 | A |
+| 3396811    | 2010-12-29 | 78 | 4 | B |
+| 0129681    | 2010-01-02 | 80 | 3 | B |
+| 9766728    | 2007-07-10 | 77 | 3 | B |
+| 6382922    | 2008-03-20 | 90 | 5 | B |
+
+## Quiz: Tidying Data
+
+Now that we've looked at several examples of messy data and ways to rearrange the data to make it tidy, let's look at a fabricated example and try to diagnose some of its problems.
+
+Take a look at the sample table provided below.  It's similar to what you might see in a publication, and it's in a great format for humans... but it's not tidy enough to work with easily in a computational way.  This table shows the results of the "QPT" psychometric (something we made up) and shares pre- and post-treatment means and standard deviations for different kinds of research cohorts.
```

Comment:
You can improve accessibility here by surrounding the full question (including table) in divs, see https://liascript.github.io/course/?https://raw.githubusercontent.com/liaScript/docs/master/README.md#67

Reply:
@rosemm I'm not sure I understand the documentation and what you're getting at -- should I move the <div = "question"> up above the problem presentation text?  (Also, I  keep trying to reply to comments and for whatever reason they aren't showing up, so if this is the third time you're seeing this, that's why....)

Reply:
@rosemm OK, for now, what I've done is move that `<div>` up -- let me know if I'm way off base, or, if you feel comfortable, just fix whatever is sub-optimal according to what you think best.  I'm done unless you tell me otherwise, and off to demystify SQL!

Reply:
Ah, sorry I didn't see your comment right away --- the idea is to enclose just the stated question (not the answer options, solution, etc.) in a div, so it will get an aria label that correctly associates the quiz question with its answer. It's only necessary when the question statement is not a single paragraph. For most questions, the quiz question will be a few sentences at most with no newlines, in which case liascript correctly associates the whole question with its answer. But when the question is more complex (as in your case where there's an example table as part of the question), we need to explicitly lump the separate pieces of the question together so they're all associated with the answer, not just the last line of it. There's no class or anything for it, just a plain little div. 
I fixed it in a commit, so I think it's all set now!